tools/eksDistroBuildToolingOpsTools/pkg/issueManager/issueManager_test.go

```diff
@@ -17,22 +17,23 @@ import (
 )
 
 const (
-	HttpOkStatusCode = 200
+	HttpOkStatusCode        = 200
 	rateLimitingIssuesError = "rate limited while attempting to create github issues"
 )
 
 var (
-	TestRepoOwner      = "TestTesterson"
+	TestRepoOwner      = "TestTestPerson"
```

Comment:
lol this wasn't a typo, I imagined someone literally named "Test Testerson" as my test case haha